source/common/stats/symbol_table_impl.h

```diff
@@ -328,6 +328,10 @@ class StatName {
     return HashUtil::xxHash64(absl::string_view(cdata, dataSize()));
   }
 
+  template <typename H> friend H AbslHashValue(H h, StatName stat_name) {
```

Comment:
Can we add a `absl::VerifyTypeImplementsAbslHashCorrectly` test for `StatName` (https://abseil.io/tips/152)?

Reply:
instead of hashing a hash, you should hash the underlying value directly

Reply:
The underlying value is a sized array of uint8_t ... are you suggesting something like this, for the StatName type:
```
template <typename H> friend H AbslHashValue(H h, StatName stat_name) {
    const char* cdata = reinterpret_cast<const char*>(stat_name.data());
    return return H::combine(std::move(h), absl::string_view(cdata, stat_name.dataSize()));
}
```

I missed the subtlety that H::combine was expecting to take a  value that can be hashed, rather than a hash.

Reply:
Yup, that is exactly what I am suggesting